src/horizontal_ie9.jsx

```diff
@@ -1,4 +1,7 @@
-import _ from 'lodash';
+import extend from 'lodash/extend';
+import range from 'lodash/range';
+import invokeMap from 'lodash/invokeMap';
+import sum from 'lodash/sum';
```

Comment:
hehe yeah seems like just npm installed it and is very well fractioned, was going to suggest to use the `per method` distributed version but this is the same and easier, cool :) 👍